src/shared/GitHub.UI/ViewModels/TwoFactorViewModel.cs

```diff
@@ -56,10 +55,13 @@ private bool CanVerify()
         public bool IsSms
         {
             get => _isSms;
-            set => SetAndRaisePropertyChanged(ref _isSms, value);
+            set
+            {
+                SetAndRaisePropertyChanged(ref _isSms, value);
+                RaisePropertyChanged(nameof(Description));
+            }
         }
 
-        [DependsOn(nameof(IsSms))]
```

Comment:
`DependsOnAttribute` is an Avalonia-specific type that is used in the Avalonia framework to implement `RaisePropertyChanged` automatically. Since we're trying to share the view models between WPF and Avalonia, we need to do things manually/without magic framework-types :)